package.json

```diff
@@ -22,7 +22,8 @@
         "npm": ">=8.1.0"
     },
     "dependencies": {
-        "fs-extra": "9.1.0"
+        "fs-extra": "9.1.0",
+        "fuse.js": "^6.5.3"
```

Comment:
Should be added as `devDependency`.
In HaruNeko, the `dependencies` are packages that are going to be installed along with the desktop client (NW.js).
The `devDependencies` will be compiled/bundled directly into the web application.

Reply:
Thank you for your kind suggestion, the proposed change has been applied at https://github.com/manga-download/haruneko/pull/37/commits/71a38ab46e4e33134c5ccf8766df3b3d90bfe9d1